src/confdb/gui/ConfigurationTreeEditor.java

```diff
@@ -134,9 +134,9 @@ public Object getCellEditorValue() {
 			EventContent content = (EventContent) toBeEdited;
 			content.setName(name);
 			treeModel.nodeChanged(content);
-			Iterator<EventContent> itC = config.contentIterator();
-			while (itC.hasNext())
-				treeModel.nodeChanged(itC.next());
+			for(int idx = 0; idx < config.contentCount(); ++idx){
+			    treeModel.nodeChanged(config.content(idx));
+			}
```

Comment:
I actually wonder why all EventContents existing in the config need to have treeModel.nodeChanged called, as opposed to just the single one being edited (in line 136). Maybe one could actually get rid of the loop over all?

Reply:
I tried, but the result is not as good graphically. Without the loop, the renamed EventContent will move to its new position (in alphabetical order), but its original slot will remain blank, and one of the other EventContents won't be shown anymore. The loop is somehow updating them all such that they are all displayed and in alphabetical order.

Reply:
Maybe get rid of line 136 only? It sets as nodeChanged the current content one and latter loops over all, so the current content one gets nodeChanged twice?

Reply:
Tried, but it doesn't do the trick.. the GUI seems to behave the same with and without L136. Trying to have a closer look.